### PR TITLE
fix: Cognite3DModel.mapFromCdfToModelCoordinates and mapPositionFromModelToCdfCoordinates correctly returns transformed coordinates

### PR DESCRIPTION
**Pull request by larsmoa**
Previously input coordinates was returned

### CODE REVIEW

File: viewer/src/public/migration/Cognite3DModel.ts [hunk 1]
```diff
@@ -135,12 +135,13 @@ export class Cognite3DModel extends THREE.Object3D implements CogniteModelBase {
    * Z-up coordinate system while ThreeJS uses a right-hand Y-up coordinate system.
    * @param p     The CDF coordinate to transform.
    * @param out   Optional preallocated buffer for storing the result.
+   * @returns Transformed position.
    */
   mapFromCdfToModelCoordinates(p: THREE.Vector3, out?: THREE.Vector3): THREE.Vector3 {
     out = out !== undefined ? out : new THREE.Vector3();
     out.copy(p);
     p.applyMatrix4(this.cadModel.modelMatrix);
```
Comment (maksnester) [inline viewer/src/public/migration/Cognite3DModel.ts:143]: here you call p.applyMatrix4

File: viewer/src/public/migration/Cognite3DModel.ts [hunk 2]
```diff
@@ -149,12 +150,13 @@ export class Cognite3DModel extends THREE.Object3D implements CogniteModelBase {
...
    * @param out     Optional preallocated buffer for storing the result.
+   * @returns Transformed position.
    */
   mapPositionFromModelToCdfCoordinates(p: THREE.Vector3, out?: THREE.Vector3): THREE.Vector3 {
-    // fixme: unused
     out = out !== undefined ? out : new THREE.Vector3();
-    p.applyMatrix4(this.cadModel.inverseModelMatrix);
-    return p;
+    out.copy(p);
+    out.applyMatrix4(this.cadModel.inverseModelMatrix);
```
Comment (maksnester) [inline viewer/src/public/migration/Cognite3DModel.ts:158]: but here you do it for out - is it correct?

**Review by maksnester — COMMENTED** [review]
Great that it got attention. 

I personally used it in that mutational way here in 3dm subapp: https://github.com/cognitedata/cdf-3d-management/blob/3f2e7ea6f4b47969bd0940c939ba9b512b6051a4/src/pages/RevisionDetails/components/ThreeDViewerToolbar/ThreeDViewerToolbar.tsx#L91-L92

Hope that fix won't break anything.

**Comment by larsmoa** [comment]
> Great that it got attention.
> 
> I personally used it in that mutational way here in 3dm subapp: https://github.com/cognitedata/cdf-3d-management/blob/3f2e7ea6f4b47969bd0940c939ba9b512b6051a4/src/pages/RevisionDetails/components/ThreeDViewerToolbar/ThreeDViewerToolbar.tsx#L91-L92
> 
> Hope that fix won't break anything.

That should be just fine :)

**Review by maksnester — APPROVED** [review]

**Pull request merged**